### PR TITLE
Add bower to npm dependencies

### PR DESCRIPTION
**Pull request by peikk0**

### CODE REVIEW

**Comment by fdubost** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
   "name": "jenkinslight",
   "version": "0.0.0",
+  "dependencies": {
+    "bower": "~1.3.9"
```
Comment (fdubost) [inline package.json:5]: Should it not be in devDependencies like grunt ?

Comment (peikk0) [inline package.json:5]: You need bower to deploy it in production (and grunt too btw).

Comment (fdubost) [inline package.json:5]: So bower and grunt should be in the same place : either in `dependencies` or in `devDependencies` :smile_cat:
But I agree with you, if we need it to deploy in prod, it should be in `dependencies`.

**Pull request merged**